src/lib/ListMetrics.js

```diff
@@ -3,33 +3,101 @@ import { errors } from 'arsenal';
 import { getMetricFromKey, getKeys, generateStateKey } from './schema';
 
 /**
-* Provides static methods to get bucket level metrics
+* Provides methods to get metrics of different levels
 */
-export default class Buckets {
+export default class ListMetrics {
+
+    /**
+     * Assign the metric property to an instance of this class
+     * @param {string} metric - The metric type (e.g., 'buckets', 'accounts')
+     */
+    constructor(metric) {
+        this.metric = metric;
+    }
+
+    /**
+     * Create the metric object to retrieve data from schema methods
+     * @param {string} value - The value of the metric type
+     * @return {object} obj - Object with a key-value pair for a schema method
+     */
+    _getSchemaObject(value) {
+        // key for the schema should not be plural metric type, so remove 's'.
+        let type;
+        if (this.metric === 'buckets') {
+            type = 'bucket';
+        } else if (this.metric === 'accounts') {
+            type = 'accountId';
+        }
+        const obj = {};
+        obj[type] = value;
+        return obj;
+    }
+
+    // Create the metric response object for a given metric.
+    _getMetricRes(value, start, end) {
+        const metricRes = {
```

Comment:
Let's move this to a JSON file. The benefit would be that it would be self-validating for syntax, ListMetrics would be smaller, the JSON would be loaded only once in the memory.  Make sure you make a deep copy of the JSON before using it.